core/rfb.js

```diff
@@ -140,16 +141,39 @@ export default function RFB(target, url, options) {
     // Bound event handlers
     this._eventHandlers = {
         focusCanvas: this._focusCanvas.bind(this),
+        windowResize: this._windowResize.bind(this),
     };
 
     // main setup
     Log.Debug(">> RFB.constructor");
 
-    // Target canvas must be able to have focus
-    if (!this._target.hasAttribute('tabindex')) {
-        this._target.tabIndex = -1;
+    // Make sure our CSS is loaded
+    const styleurl = "core/rfb.css";
+    let links = document.getElementsByTagName('link');
+    let found = false;
+    for (let i = 0; i < links.length; i++) {
+        if (links[i].getAttribute('href') === styleurl) {
+            found = true;
+            break;
+        }
+    }
+    if (!found) {
+        let link = document.createElement('link');
+        link.rel = "stylesheet";
+        link.href = styleurl;
+        document.head.appendChild(link);
     }
 
+    // Create DOM elements
+    this._screen = document.createElement('div');
+    this._screen.className = 'noVNC_screen';
```

Comment:
Sure. Suggestions? `noVNC_container`?

Reply:
The div with id `noVNC_screen` is inside a div called `noVNC_container` in vnc.html. So that name is also taken. Maybe `noVNC_canvas_container`? But I don't really have any good suggestions

Reply:
I found a simple solution; `#noVNC_screen` is no longer needed so I just removed it. :)

Reply:
great!